presto-main/src/test/java/com/facebook/presto/block/TestInt128ArrayBlock.java

```diff
@@ -82,6 +85,19 @@ public void testCompactBlock()
         testIncompactBlock(new Int128ArrayBlock(valueIsNull.length - 2, Optional.of(valueIsNull), longArray));
     }
 
+    @Test
+    public void testIsDistinctFrom()
+    {
+        long[] arr1 = {112L, 0L}; // DECIMAL '1.12'
```

Comment:
- don't use a1, a2, etc. variables names; here, I'd just inline arr1 and arr2
- static import distinctBlockPositionLongLong

```
    @Test
    public void testIsDistinctFrom()
    {
        Block left = new Int128ArrayBlock(1, Optional.empty(), new long[] {112L, 0L});
        Block right = new Int128ArrayBlock(1, Optional.empty(), new long[] {185L, 0L});

        assertFalse(distinctBlockPositionLongLong(left, 0, left, 0));
        assertTrue(distinctBlockPositionLongLong(left, 0, right, 0));
    }
```